### PR TITLE
Webpack create robots txt

### PR DESCRIPTION
**Pull request by froi**
Why

Only our main production site should be index on the web. Our staging and dev sites should remain under the radar when it comes to search.

What Changed

- Add robots.txt webpack plugin that will create the proper robots.txt for the deployment environment.- 12ab251
- Add robots.txt webpack plugin config - e6c1c2f
- Remove config/webpack.github-deploy.js - 31ae7b9

### CODE REVIEW

**Comment by froi** [comment]
[Prevent Staging from being indexed #Sprint_17 #Sprint_18](https://trello.com/c/y9DXWWb8/338-prevent-staging-from-being-indexed-sprint17-sprint18)

File: config/webpack.federalist-deploy.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ const PreloadWebpackPlugin = require('preload-webpack-plugin');
 const webpackMerge = require('webpack-merge'); // used to merge webpack configs
 const ghpages = require('gh-pages');
 const webpack = require('webpack');
+const RobotstxtPlugin = require('robotstxt-webpack-plugin').default;
```
Comment (zachary-kuhn) [inline config/webpack.federalist-deploy.js]: Cool, didn't even think about using a plugin for this.

File: config/webpack.federalist-deploy.js [hunk 2]
```diff
@@ -144,14 +145,14 @@ module.exports = function (env) {
...
 
-          ghpages.publish(webpackConfig({env: ENV}).output.path, options, function(err) {
-            if (err) {
-              console.log('GitHub deployment done. STATUS: ERROR: '+err);
-              throw err;
-            } else {
-              console.log('GitHub deployment done. STATUS: SUCCESS.');
-            }
-          });
+          // ghpages.publish(webpackConfig({env: ENV}).output.path, options, function(err) {
```
Comment (zachary-kuhn) [inline config/webpack.federalist-deploy.js]: We still need this deploy for federalist, right?

File: config/webpack.github-deploy.js [hunk 3]
```diff
@@ -1,137 +0,0 @@
-/**
```
Comment (zachary-kuhn) [inline config/webpack.github-deploy.js]: Bravo!

File: config/webpack.federalist-deploy.js (hunk 2 above)
Comment (froi) [reply config/webpack.federalist-deploy.js]: Damn. I forgot I commented that. Changing now.

**Pull request merged**